.github/workflows/run-build-and-test.yml

```diff
@@ -0,0 +1,37 @@
+# This workflow will do a clean installation of node dependencies, cache/restore them, build the source code and run tests across different versions of node
+# For more information see: https://docs.github.com/en/actions/automating-builds-and-tests/building-and-testing-nodejs
+
+name: Run build and test
+
+on:
+  push:
+    branches: [ "main" ]
+  pull_request:
+    branches: [ "main" ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [16.x]
+        # See supported Node.js release schedule at https://nodejs.org/en/about/releases/
+
+    steps:
+    - uses: actions/checkout@v3
+    - name: Setup pnpm
+      # You may pin to the exact commit or the version.
+      # uses: pnpm/action-setup@d882d12c64e032187b2edb46d3a0d003b7a43598
+      uses: pnpm/action-setup@v2.4.0
+      with:
+        version: 8.6.10
+    - name: Use Node.js ${{ matrix.node-version }}
+      uses: actions/setup-node@v3
+      with:
+        node-version: ${{ matrix.node-version }}
+        cache: 'pnpm'
+    - run: pnpm install
+    - run: pnpm build
+    - run: pnpm test
```

Comment:
i dont think we need `pnpm test` here

Reply:
The main purpose of this workflow is to check if all the tests are passed and the build can be run successfully with the changes of the PR, so I think it's necessary to have `pnpm test` here 🤔

Reply:
I dont say that we dont need `pnpm test`. Just want to move it to another job

Reply:
Updated in [f79eed4](https://github.com/chalalala/remo/pull/10/commits/f79eed484fc05ff743697b8a3424c060d4df8c92)